src/main/kotlin/com/wutsi/blog/client/user/SearchUserRequest.kt

```diff
@@ -8,5 +8,6 @@ data class SearchUserRequest(
         val offset: Int = 0,
         val blog: Boolean? = null,
         val sortBy: UserSortStrategy = UserSortStrategy.created,
-        val sortOrder: SortOrder = SortOrder.ascending
+        val sortOrder: SortOrder = SortOrder.ascending,
+        val autoFollowedByBlogs: Boolean? = null
```

Comment:
- set this by default to false
- Please to the same for `blog`
```
  val blog: Boolean = false
  val autoFollowedByBlogs: Boolean = false
```

Reply:
done